app/components/user-login/via-sso/index.hbs

```diff
@@ -0,0 +1,59 @@
+{{#if @isEnforced}}
+  <form ...attributes onsubmit='return false' class='w-full'>
+    <AkStack
+      @width='full'
+      @direction='column'
+      @alignItems='stretch'
+      @spacing='3'
+    >
+      <AkTextField
+        @label={{t 'usernameEmailIdTextLabel'}}
+        @placeholder={{t 'usernameEmailIdTextPlaceholder'}}
+        @labelTypographyColor='textPrimary'
+        @labelTypographyVariant='body2'
+        @textFieldRootClass={{@textFieldClass}}
+        @value={{@username}}
+        {{on 'input' @usernameChanged}}
+        autocomplete='username'
+        data-test-user-login-via-sso-forced-username-input
+      />
+
+      <AkButton
+        {{on 'click' @ssologin}}
+        @type='submit'
+        @loading={{@showSSOSpinner}}
+        {{autofocus}}
+        data-test-user-login-via-sso-forced-button
+      >
+        {{t 'ssoLogin'}}
+      </AkButton>
+    </AkStack>
+  </form>
+{{else}}
+  <UserLogin::ViaUsernamePassword
+    @username={{@username}}
+    @password={{@password}}
+    @usernameChanged={{@usernameChanged}}
+    @showSpinner={{@showLoginSpinner}}
+    @login={{@login}}
+    @textFieldClass={{@textFieldClass}}
+    @showCredError={{@showCredError}}
+    @showAccountLockError={{@showAccountLockError}}
+    data-test-user-login-via-sso-not-forced-login
+  />
+
+  <div data-test-user-login-via-sso-or local-class='login-form-sso-separator'>
+    <div local-class='login-form-sso-separator-or'>
+      {{t 'or'}}
+    </div>
+  </div>
```

Comment:
this can be done with `ak-stack`, `ak-divider` & `ak-typography` without requiring additional css